models/repo.go

```diff
@@ -277,6 +277,53 @@ func (repo *Repository) APIFormat(mode AccessMode) *api.Repository {
 	return repo.innerAPIFormat(x, mode, false)
 }
 
+// GqlFormat converts a Repository to api.GqlRepository
+func (repo *Repository) GqlFormat(mode AccessMode) *api.GqlRepository {
```

Comment:
I would add this FormatConvertion into the Convert package

Reply:
I put it here because the "APIFormat" function was already here that the REST Api uses and it seemed to make sense to have them close together. Do you still think it should move?

Reply:
@bhalbright they where but migration already started, `func (repo *Repository) APIFormat()` has to be moved too ...

Reply:
I moved it, thanks!